sensor/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/board_detectors/board_detector_factory.py

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
+# cSpell:enableCompoundWords
 from intrinsic_camera_calibrator.board_detectors.apriltag_grid_detector import ApriltagGridDetector
```

Comment:
Considering http://wiki.ros.org/apriltag_ros, we can add `apriltag` to the dictionary.

Reply:
Added it to the respective PR